proposals/NNNN-DisabledSoftbuttons.md

```diff
@@ -0,0 +1,89 @@
+
+# Disabled Softbuttons
+
+* Proposal: [SDL-NNNN](NNNN-ButtonIdentifiers.md)
+* Author: [Michael Crimando](https://github.com/MichaelCrimando)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Java Suite / Web / RPC ]
+
+## Introduction
+
+This feature would allow apps to send custom softbuttons to display on screen but they would appear disabled.
+Also this allows an app to subscribe to softbuttons that are disabled
+
+## Motivation
+Some apps like iHeartRadio don't provide the same set of custom softbuttons for each home screen.
+This would allow them to display that to the user, just like the native app on the phone.  Other apps like Pandora only support Seek Left and Repeat if the user has a premium account. This can let them show the buttons and let the user know it's available just not right now.
+
+## Proposed solution
+
+For custom softbuttons, add a parameter to `SoftButtonCapabilities` to let the app knows that the ability is possible.
+```xml
+<struct name="SoftButtonCapabilities">
+    .
+    .
+    .
+    <param name="disabledButtonPossible" type="Boolean" mandatory="true">
+        <description>The button supports being disabled. If the button is set to disabled, it will still show on the HMI but appear grayed out
```

Comment:
```suggestion
        <description>The button supports being disabled. If the button is set to disabled, it will still show on the HMI but appear grayed out.
```